utilities/ifix-migration-toolkit/mvnw

```diff
@@ -0,0 +1,316 @@
+#!/bin/sh
```

Comment:
Delete this file. It is unnecessary

Reply:
minor change, to be taken up as part of code cleanup.